README.md

```diff
@@ -25,7 +25,7 @@ You want grammY because it is easy to use. It is very powerful and always up to
 
 Are you ready? 🤖🚀
 
-Bots are written in [TypeScript](https://www.typescriptlang.org/) (or JavaScript) and run on [Node.js](https://nodejs.org/).
+Bots are written in [TypeScript](https://www.typescriptlang.org/) (or JavaScript) and run on [Node.js](https://nodejs.org/) and on [Deno](#deno-support).
```

Comment:
Maybe switch Node.js and Deno to highlight Deno as "the first one"?

Reply:
We can do this when Deno overtakes Node. For now, I'd mention it, but since most users are running Node, I'd list it first.

Reply:
My idea behind it: grammY is written for Deno which also supports Node.js. Users using Node.js will also find Node.js just two words later. People interested might test out Deno because its "the first". People not interested will use what they already know which is Node.js.

Reply:
The idea is to develop something for Deno because that's what we like, and to ship something for Node because that's what everyone else likes (according to current numbers). When Deno overtakes Node, we can switch like nothing ever happened.